src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -291,7 +291,7 @@ internal async Task ResolveAuthorityEndpointsAsync()
             IDictionary<string, string> additionalBodyParameters,
             CancellationToken cancellationToken)
         {
-            OAuth2Client client = new OAuth2Client(ServiceBundle.DefaultLogger, ServiceBundle.HttpManager, ServiceBundle.TelemetryManager);
+            OAuth2Client client = new OAuth2Client(ServiceBundle.DefaultLogger, ServiceBundle.HttpManager, ServiceBundle.TelemetryManager, ServiceBundle.PlatformProxy.CreateDeviceAuthManager());
```

Comment:
Why don't you expose `ServiceBundle.DeviceManager` like all the other objects in ServiceBundle? The ServiceBundle's lifetime is the same as the `pca` / `cca` objects, which seems better suited for DeviceManager. I do not see a need to have a new object like this for every request.